### PR TITLE
Bottomsheet improvements

### PR DESCRIPTION
**Pull request by saket**

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/widgets/BottomSheetActivity.kt [hunk 1]
```diff
@@ -2,19 +2,55 @@ package org.simple.clinic.widgets
...
     super.finish()
     overridePendingTransition(0, 0)
   }
+
+  override fun setContentView(layoutResId: Int) {
+    LayoutInflater.from(this).inflate(layoutResId, contentContainer)
+  }
+
+  override fun setContentView(view: View?) {
+    throw AssertionError()
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/widgets/BottomSheetActivity.kt]: It's not clear why the assertion error is being thrown.

File: app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntrySheet.kt [hunk 2]
```diff
@@ -61,7 +61,11 @@ class CustomPrescriptionEntrySheet : BottomSheetActivity() {
   }
 
   override fun onBackgroundClick() {
-    if (drugNameEditText.text.isBlank() && drugDosageEditText.text.isBlank()) {
+    val drugNameEmpty = drugNameEditText.text.isEmpty()
+    val dosageEmpty = drugDosageEditText.text.isEmpty()
+        || drugDosageEditText.text.toString() == getString(R.string.customprescription_dosage_placeholder)
```
Comment (vinaysshenoy) [inline app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntrySheet.kt]: Do we need to ignore whitespaces when checking for the placeholder string?

Comment (saket) [reply app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntrySheet.kt]: Done

File: app/src/main/java/org/simple/clinic/widgets/BottomSheetActivity.kt (hunk 1 above)
Comment (saket) [reply app/src/main/java/org/simple/clinic/widgets/BottomSheetActivity.kt]: Fixed

**Pull request merged**